docs/instrumentation-list.yaml

```diff
@@ -4742,8 +4747,11 @@ libraries:
       - org.hibernate.reactive:hibernate-reactive-core:(,)
   hikaricp:
   - name: hikaricp-3.0
+    display_name: HikariCP
     description: |
-      This instrumentation provides database client metrics for HikariCP 3.0+ connection pools. It reports metrics like connection timeouts, creation time, wait time, and usage time, along with connection pool statistics such as the number of active, idle, and pending connections.
+      This instrumentation generates database client metrics for HikariCP 3.0+ connection pools. It reports metrics like connection timeouts, creation time, wait time, and usage time, along with connection pool statistics such as the number of active, idle, and pending connections.
```

Comment:
do other instrumentations also include versions like `3.0+` in the description?

Reply:
no, as a general rule i've been trying to only include it if there is some important distinction involved with the versions, but in this case it doesn't, so i'll remove it